docs/reference/search/multi-search.asciidoc

```diff
@@ -121,6 +121,18 @@ conditions is met:
   - The request targets one or more read-only index.
   - The primary sort of the query targets an indexed field.
 
+`recall_goal`::
+(Optional, integer)
+An aid for "query relaxation". If > 0 then requests are processed sequentially
+until a search produces the required minimum number of matches (subsequent requests
```

Comment:
May be, make it more clear that `recall_goal` applies to each individual search, not a cumulative (as I similar to @romseygeek  concluded from this documentation).